tests/telemetry_test.py

```diff
@@ -987,437 +948,179 @@ def test_missing_properties_shard_stats(self, metrics_store_put_doc):
-                        "count": 0,
-                        "sum": 0,
-                        "min": 0,
-                        "max": 0
+                    "direct_bytes_read": {"count": 0, "sum": 0, "min": 0, "max": 0, "time": "0s", "time_in_nanos": 0},
+                    "optimized_bytes_read": {"count": 0, "sum": 0, "min": 0, "max": 0, "time": "0s", "time_in_nanos": 0},
+                    "forward_seeks": {
+                        "small": {"count": 0, "sum": 0, "min": 0, "max": 0},
+                        "large": {"count": 0, "sum": 0, "min": 0, "max": 0},
                     },
-                    "large": {
-                        "count": 0,
-                        "sum": 0,
-                        "min": 0,
-                        "max": 0
-                    }
-                },
-                "blob_store_bytes_requested": {
-                    "count": 50,
-                    "sum": 279900,
-                    "min": 5598,
-                    "max": 5598
-                },
-                "current_index_cache_fills": 0
-            },
-            {
-                "file_ext": "kdd",
-                "num_files": 50,
-                "total_size": 356841728759,
-                "open_count": 174,
-                "close_count": 174,
-                "contiguous_bytes_read": {
-                    "count": 184852,
-                    "sum": 189288448,
-                    "min": 1024,
-                    "max": 1024
-                },
-                "non_contiguous_bytes_read": {
-                    "count": 2228,
-                    "sum": 2281472,
-                    "min": 0,
-                    "max": 1024
-                },
-                "cached_bytes_read": {
-                    "count": 187049,
-                    "sum": 191538176,
-                    "min": 1024,
-                    "max": 1024
-                },
-                "index_cache_bytes_read": {
-                    "count": 31,
-                    "sum": 31744,
-                    "min": 0,
-                    "max": 1024
-                },
-                "cached_bytes_written": {
-                    "count": 122,
-                    "sum": 274173997,
-                    "min": 1024,
-                    "max": 7942949,
-                    "time": "22.2s",
-                    "time_in_nanos": 22277973991
-                },
-                "direct_bytes_read": {
-                    "count": 0,
-                    "sum": 0,
-                    "min": 0,
-                    "max": 0,
-                    "time": "0s",
-                    "time_in_nanos": 0
-                },
-                "optimized_bytes_read": {
-                    "count": 0,
-                    "sum": 0,
-                    "min": 0,
-                    "max": 0,
-                    "time": "0s",
-                    "time_in_nanos": 0
-                },
-                "forward_seeks": {
-                    "small": {
```

Comment:
Can we please keep the current formatting here? Similarly for all nested structures below.